insolar/network.go

```diff
@@ -47,6 +47,8 @@ type Network interface {
 	Leave(ctx context.Context, ETA PulseNumber)
 	// GetState returns our current thoughs about whole network
 	GetState() NetworkState
+	// SetOperableFunc registers callback for notifying of network state
+	SetOperableFunc(f func(ctx context.Context, operable bool))
```

Comment:
If we want to notify other insolar components of network state change, we should pass state instead of a boolean flag that has uncertain nature. The name of the callback also should be picked more carefully

Reply:
No. We not need to force users to understand what we allowing in what state.